test/test_conflictingdeps.py

```diff
@@ -0,0 +1,66 @@
+from .harness import Harness
+import uuid
+import pytest
+
+@pytest.fixture()
+def workspace():
+    workspace = Harness("repos/confl_dep")
+    workspace.initialize("repos/confl_dep?" + str(uuid.uuid4()))
+    yield workspace
+    workspace.exit()
+
+class TestConflictingDependencies:
+    def test_hover_packages_conflicting_module_names(self, workspace):
+        uri = "file:///confl_dep/__main__.py"
+    
+        # django import
+        d_line, d_col = 0, 11
+        d_result = workspace.hover(uri, d_line, d_col)
+        assert d_result == {
+            'contents': [
+                {
+                    'language': 'python', 
+                    'value': 'django'
+                }
+            ]
+        }
+
+        # jinja2 import
+        j_line, j_col = 1, 11
+        j_result = workspace.hover(uri, j_line, j_col)
+        assert j_result == {
+            'contents': [
+                {
+                    'language': 'python', 
+                    'value': 'jinja2'
+                }, 
+                'jinja2\n'
```

Comment:
Not sure, but it may make sense to use a multi-line string here?

Reply:
This one I just coped from the test's output after verifying that it was correct. I'm going to merge this without transforming this / aligning it into a multiline string. If you feel strongly about it, let me know and I'll look at it again.